manifests/base/params.env

```diff
@@ -16,3 +16,4 @@ odh-tensorflow-gpu-notebook-image-n-2=quay.io/modh/cuda-notebooks@sha256:6fadedc
 odh-trustyai-notebook-image-n=quay.io/modh/odh-trustyai-notebook@sha256:b68c1bfd9926b224180835382b36ad25e2269ffb95fca0646a89c8cceb6a6e7a
 odh-trustyai-notebook-image-n-1=quay.io/modh/odh-trustyai-notebook@sha256:de46659eccc3dd8e4b79cfbf7fc95464f76ffcc06f3f914841533130fba2985f
 odh-habana-notebook-image-n=quay.io/modh/odh-habana-notebooks@sha256:0f6ae8f0b1ef11896336e7f8611e77ccdb992b49a7942bf27e6bc64d73205d05
+odh-codeserver-notebook-n=quay.io/modh/codeserver@sha256:
```

Comment:
I suppose that this is something that is gonna be updated later, right?

Reply:
Exactly, i will open a follow up PR to fill this value. Once this PR get merged the CI will build the image and eventually I will be able to use it on the params.env